src/spectrecoff/Output.fs

```diff
@@ -210,4 +210,4 @@ let toConsole (payload: OutputPayload) =
 type OutputPayload with
     member self.toMarkedUpString = toMarkedUpString self
     member self.toRenderable = payloadToRenderable self
-    member self.toConsole = toConsole self
+    member self.toConsole = toConsole self
```

Comment:
I'd prefer not to add this extension method.

Reply:
Probably the changes view mislead you a bit. 

I didn't add any methods here.

What happened is my IDE automatically removed the empty row at the end of the file. 

Would you prefer to revert that empty line?

Reply:
Oh sorry, my bad. Nah, it's fine.